examples/allow_js/BUILD.bazel

```diff
@@ -37,5 +39,27 @@ write_source_files(
         "expected.jsx": "js_outs/in.js",
         "expected.mjs": "js_outs/in.mjs",
         "expected.cjs": "js_outs/in.cjs",
+        "expected.ts": "js_outs/in_ts.js",
     },
 )
+
+swc(
+    name = "compile-js-exts-no-js",
+    srcs = [
+        "in.cjs",
+        "in.d.ts",
+        "in.jsx",
+        "in.mjs",
+        "in_ts.ts",
+    ],
+    out_dir = "no_js_outs",
+    allow_js = False,
+)
+
+assert_outputs(
+    name = "test-x-no-js",
```

Comment:
What is this actually testing? It seems to just be asserting that a `.ts` got transpiled, nothing asserting "no js"... 🤔

Reply:
Without `allow_js = False`, this test fails:

```
==================== Test output for //examples/allow_js:test-x-no-js:
0a1,3
> examples/allow_js/no_js_outs/in.cjs
> examples/allow_js/no_js_outs/in.js
> examples/allow_js/no_js_outs/in.mjs
FAIL: files "examples/allow_js/_expected_test-x-no-js.txt" and "examples/allow_js/_test-x-no-js_outputs.txt" differ. 
```